aws/resource_aws_vpn_connection.go

```diff
@@ -495,10 +977,273 @@ func resourceAwsVpnConnectionRead(d *schema.ResourceData, meta interface{}) erro
 func resourceAwsVpnConnectionUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).ec2conn
 
+	tun1Changed := false
```

Comment:
This section could be broken up into a few smaller functions to help with readability also.

Reply:
Done